bittensor/_synapse/__init__.py

```diff
@@ -224,5 +226,7 @@ def deserialize( synapse_wire_proto: bittensor.proto.Synapse ) -> Synapse:
             return TextCausalLMNext.deserialize_from_wire_proto(synapse_wire_proto)
         elif synapse_wire_proto.synapse_type == bittensor.proto.Synapse.SynapseType.TEXT_SEQ_2_SEQ:
             return TextSeq2Seq.deserialize_from_wire_proto( synapse_wire_proto )
+        elif synapse_wire_proto.synapse_type == bittensor.proto.Synapse.SynapseType.TEXT_MASKED_LAST_HIDDEN_STATE:
```

Comment:
When do we call TEXT_MASKED_LAST_HIDDEN_STATE? Is this a seperate synapse or a special case of the TEXT_LAST_HIDDEN_STATE?

Reply:
removed this.